compiler/options.nim

```diff
@@ -234,6 +234,7 @@ type
                              ## symbols are always guaranteed to be style
                              ## insensitive. Otherwise hell would break lose.
     packageCache*: StringTableRef
+    nimblePaths*: seq[AbsoluteDir]
```

Comment:
This already exists by the name of ``lazyPaths``, why can't we use that instead?

Reply:
`lazyPaths` includes the nimbleDir as well as every pkg dir discovered under it. You need the base dir for $nimbleDir substitution and that isn't being saved today.

https://github.com/nim-lang/Nim/blob/e5478b32a891077972c53b0715481fb7c6b4390c/compiler/nimblecmd.nim#L113